docs/md/directory-usage.md

```diff
@@ -0,0 +1,126 @@
+# Using WFL Across a Directory
+
+Both the [WGS](../modules-wgs/) and [XX](../modules-external-exome-reprocessing/) 
+modules support starting workflows from a single file each--the modules are
+capable of extrapolating the other required inputs.
+
+If you have a set of files uploaded to a GCS bucket and you'd like to start
+a workflow for each one, you can do that via shell scripting.
+
+!!! warning "Note"
+    You will likely run into performance issues with WFL if you try to start
+    hundreds or thousands of workflows in a single request to WFL. You'll need
+    to split up the workflows into multiple workloads, tips for that are
+    [here](#other-notes).
+
+## Step 1: List Files
```

Comment:
Make this a worked example. State what we're trying to do and then give the steps for how to do it:

"Let's say you wan to reprocess whole genomes. To do this, you want workflow-launcher to launch a "ExternalWhoelGenomeReprocessing" pipeline on each of the CRAMs in bucket "foo".
Step 1:

Step 2:

Step 3:

etc